README.md

```diff
@@ -1,15 +1,23 @@
 # No LinkedIn Profile Stats
 
 ## Abstract
-This browser extension hides the page view and post impression statistics on LinkedIn.
+
+This browser extension hides the following on LinkedIn:
```

Comment:
I updated this to line up with the changes I've introduced in the three PR's over the past several months.